### ISSUE TITLE
Do not unset CPT named "product"

### ISSUE THREAD

**Issue by slaxxarn** [elementor/elementor#8160]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
Was trying to create a single template for my CPT called "product", and couldn't see the CPT in the list. Did a search through the code and saw that elementor unset the post type called "product" and there is no filter or anything to enable the use of the CPT.

## Steps to reproduce

Create a cpt called "product" and try to use it for single templates.

## Isolating the problem
- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.14.2
	MySQL version:  v
	PHP Version: 7.3.5
	PHP Max Input Vars: 1000
	PHP Max Post Size: 800M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.2
	Site URL: http://isaksson.slaxxarn
	Home URL: http://isaksson.slaxxarn
	WP Multisite: No
	Max Upload Size: 800 MB
	Memory limit: -1
	Permalink Structure: /%postname%/
	Language: sv-SE
	Timezone: Europe/Stockholm
	Debug Mode: Active

== Theme ==
	Name: Twenty Nineteen
	Version: 1.4
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: sv_SE
	User Agent: Mozilla/5.0 (X11; Linux x86_64; rv:66.0) Gecko/20100101 Firefox/66.0

== Active Plugins ==
	Elementor
		Version: 2.5.15
		Author: Elementor.com

	Elementor Pro
		Version: 2.5.8
		Author: Elementor.com

	Isakssons kalkylator
		Version: 1.2.2
		Author: Dalnix AB


== Log ==
	: 
PHP: showing 1 of 1PHP: 2019-05-20 11:36:37 [notice X 1][/home/christian/projekt/isaksson/www/wp-content/plugins/elementor/includes/settings/system-info/classes/server.php::238] Trying to get property 'Value' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				
```
</details>

**Comment by LeodanDesign** [comment]
Have you had WooCommerce installed at any point? Could this be an issue with Elementor associating the "product" cpt only with WooCommerce?

**Comment by slaxxarn** [comment]
@LeodanDesign see file elementor-pro/modules/theme-builder/module.php on line 194
And to answer your question, woocommerce haven't been installed at any point but this is most likely a quickfix for a woocommerce problem you guys have had in the past.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.5.10

Feel free to update

Thanks!
